docker/docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+version: "3.5"
+networks:
+  default:
+    name: zkevm
+
+services:
+  zkevm-mock-l1-network:
+    container_name: zkevm-mock-l1-network
+    image: ethereum/client-go
+    ports:
+      - 8545:8545
+      - 8546:8546
+    command:
+      - "--http"
+      - "--http.api"
+      - "admin,eth,debug,miner,net,txpool,personal,web3"
+      - "--http.addr"
+      - "0.0.0.0"
+      - "--http.corsdomain"
+      - "*"
+      - "--http.vhosts"
+      - "*"
+      - "--ws"
+      - "--ws.origins"
+      - "*"
+      - "--ws.addr"
+      - "0.0.0.0"
+      - "--dev"
+      - "--datadir"
+      - "/geth_data"
+      - "--syncmode"
+      - "full"
+      - "--rpc.allow-unprotected-txs"
+
+  zkevm-contracts:
+    container_name: zkevm-contracts
+    image: snapchain/zkevm-contracts
+    depends_on:
+      zkevm-mock-l1-network:
+        condition: service_started    
+    environment:
+      - JSONRPC_HTTP_URL=http://zkevm-mock-l1-network:8545
+      - L2_CHAIN_ID=2955
+    command:
+      - "./docker/scripts/deploy-contracts.sh"
```

Comment:
let's move this to `docker/Dockerfile` since this is likely to be the only use case of the docker image